07_Train_With_AML_Pipeline.ipynb

```diff
@@ -636,17 +641,141 @@
    },
    "outputs": [],
    "source": [
-    "aad_token = auth.get_authentication_header()\n",
-    "rest_endpoint = published_pipeline.endpoint\n",
-    "print(\"You can perform HTTP POST on URL {} to trigger this pipeline\".format(rest_endpoint))\n",
-    "response = requests.post(rest_endpoint, \n",
-    "                         headers=aad_token, \n",
-    "                         json={\"ExperimentName\": experiment_name,\n",
-    "                               \"RunSource\": \"SDK\"})\n",
-    "run_id = response.json()[\"Id\"]\n",
-    "print(run_id)"
+    "# aad_token = auth.get_authentication_header()\n",
+    "# rest_endpoint = published_pipeline.endpoint\n",
+    "# print(\"You can perform HTTP POST on URL {} to trigger this pipeline\".format(rest_endpoint))\n",
+    "# response = requests.post(rest_endpoint, \n",
+    "#                          headers=aad_token, \n",
+    "#                          json={\"ExperimentName\": experiment_name,\n",
+    "#                                \"RunSource\": \"SDK\"})\n",
+    "# run_id = response.json()[\"Id\"]\n",
+    "# print(run_id)"
```

Comment:
This section should not be commented for the master branch. Commenting is OK for an instrument branch.